Motorcycle.js

```diff
@@ -1,6 +1,31 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
+function Motorcycle(model, year, color, seats, passengers) {
+	Car.call(this, model, year, color, seats, passengers);
+	this.seats = 2;
```

Comment:
You can also use an if statement here to override user input if the number of seats is greater than 2:
```
if (this.seats > 2) {
    this.seats = 2;
};
```